src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const NavBar: React.FC = () => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
```

Comment:
better to make separate component for this link

Reply:
Agree, move it to separate component, only title and `to` is different here